libs/barista-components/sunburst-chart/src/sunburst-chart.spec.ts

```diff
@@ -225,15 +217,15 @@ describe('DtSunburst', () => {
 
       it('should clean selection on click outside', () => {
         const event = { stopPropagation: () => {} } as MouseEvent;
-        component.onClick(event);
+        component._onClick(event);
```

Comment:
Avoid this internal here as well, please create a click event - e.g. `dispatchFakeEvent(document, 'click');`